### DISCUSSION TITLE
How do you change the dictionary?

### DISCUSSION THREAD

**Discussion by Adrakaris** [psoukie/zipchord#163] [Q&A] [answered]
I've been unable to find in the documentation or in the UI any way to change the dictionary from the starting one. How do you change it to the regular qwerty dictionary?

Edit: actually, the two files seem to be the same... still, my question still stands; How do you change which dictionary files zipchord uses?

**Comment by phantom-Orange** [comment, marked as answer]
you use the `open` button, and then select the file that you want to use.

**Reply by Adrakaris** [reply]
Ah, so that's what it is. Thanks
